### PR TITLE
update docs/developers.md to reflect the new parameter changes

### PR DESCRIPTION
**Pull request by johnkim76**

### CODE REVIEW

**Comment by johnkim76** [comment]
[Document new parameter schema way of handling parameters in the ansible playbook bundle. This is for APB developers.](https://trello.com/c/nzDFvGrN/290-document-new-parameter-schema-way-of-handling-parameters-in-the-ansible-playbook-bundle-this-is-for-apb-developers)

File: docs/developers.md [hunk 1]
```diff
@@ -25,54 +25,112 @@ my-apb/
...
+        value: '{{ mariadb_name }}'
+      - name: MYSQL_USER
+        value: '{{ mariadb_user }}'
+      - name: MYSQL_PASSWORD
+        value: '{{ mariadb_password }}'
+```
+
+The above expects the `namespace` variables to be defined, which was not part of the `parameters` in the spec file `apb.yml`.
+
+To define variables, use the `main.yml` file under the `defaults` folder to define/set other variables for your APB.  For example, below is the [defualts/main.yml](https://github.com/fusor/apb-examples/blob/master/etherpad-apb/roles/provision-etherpad-apb/defaults/main.yml) for the `etherpad-apb`:
```
Comment (rthallisey) [inline docs/developers.md]: s/```[defualts/main.yaml]```/```[defaults/main.yaml]```/

File: docs/developers.md [hunk 2]
```diff
@@ -25,54 +25,112 @@ my-apb/
...
+        value: '{{ mariadb_root_password }}'
+      - name: MYSQL_DATABASE
+        value: '{{ mariadb_name }}'
+      - name: MYSQL_USER
+        value: '{{ mariadb_user }}'
+      - name: MYSQL_PASSWORD
+        value: '{{ mariadb_password }}'
+```
+
+The above expects the `namespace` variables to be defined, which was not part of the `parameters` in the spec file `apb.yml`.
```
Comment (rthallisey) [inline docs/developers.md]: s/variables/variable/

**Review by rthallisey — CHANGES_REQUESTED** [review]

**Comment by johnkim76** [comment]
@rthallisey I've made the changes and squashed

**Review by rthallisey — APPROVED** [review]

**Pull request merged**